internal/xds/bootstrap/bootstrap.go

```diff
@@ -42,10 +42,9 @@ const (
 	// DefaultWasmServerPort is the default listening port of the wasm HTTP server.
 	wasmServerPort = 18002
 
-	envoyReadinessAddressv4 = "0.0.0.0"
-	envoyReadinessAddressv6 = "::"
+	EnvoyStatsPort = 19001
 
-	EnvoyReadinessPort = 19001
+	EnvoyReadinessPort = 19003
```

Comment:
should this be 19002 or 19003 ?
cc @envoyproxy/gateway-maintainers

Reply:
19002 is used by shutdown manager